site/content/en/docs/Reference/API/Generators/ConfigMapGenerator.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "ConfigMapGenerator"
+linkTitle: "ConfigMapGenerator"
+weight: 1
+date: 2023-11-16
+description: >
+  Generate ConfigMap objects.
```

Comment:
I noticed this description is being rendered on both the parent page and in this page as well. I don't want to hold up this PR on that, but is it fair to create an issue to address it separately?

If we want to copy the behaviour from [docs.k8s.io](https://docs.k8s.io), this description would render solely on the parent page and never in the body of the page that defines it.

Reply:
Thanks for catching this! I was curious about how this is configured. I created this issue to remove the description from the body of the page: https://github.com/kubernetes-sigs/kustomize/issues/5488